railties/lib/rails/generators/rails/app/templates/config/initializers/cookies.rb.tt

```diff
@@ -3,3 +3,7 @@
 # Specify a serializer for the signed and encrypted cookie jars.
 # Valid options are :json, :marshal, and :hybrid.
 Rails.application.config.action_dispatch.cookies_serializer = :json
+
+# Specify the SameSite level protection for the cookies
+# Valid options are :no_protection, :lax, and :strict.
+Rails.application.config.action_dispatch.cookies_same_site_protection = :no_protection
```

Comment:
I think this config should instead be live in environment configuration or application configuration.

We can, say, have the default set to `:no_protection` by default, but then set this to `:lax` in production.rb template, and in the new framework default.